src/scss/components/image-fundation.scss

```diff
@@ -0,0 +1,15 @@
+.image-fundation {
+  &__picture {
+    display: flex;
+    justify-content: center;
+    overflow: hidden;
+    width: 100%;
+
+    img {
+      height: 100%;
+      min-height: 400px;
+      min-width: 700px;
```

Comment:
These properties are not needed. Just set the image width and height to 100% so that it inherits the container dimensions. Also, add the property `object-fit: cover;` so that the original image fit inside the container dimensions.

Reply:
![image](https://user-images.githubusercontent.com/65635471/92342571-d7801200-f086-11ea-8af6-b94c4d97392e.png)

When the minimum sizes are removed, it breaks up and the image remains in this way,
the object-fit property: cover does not show the expected results

Reply:
It's because you need to set the fixed height in the parent, not in the `<img>` element. 

For the `<img>`:
`width: 100%;`
`height: 100%;`
`object-fit: cover;`
Not need for a min-width o min-height here.

For the parent element of the `<img>`:
`width: 100%;`
`height: 510px;` (mobile)
`height: 620px;` (desktop)
Only these three properties, the other properties like `display: flex`, `justify-content: center;`, and `overflow: hidden;` can be removed.